terraform/modules/ip_addresses/azure_fixngo.tf

```diff
@@ -42,11 +42,13 @@ locals {
     noms_mgmt_vnet            = "10.102.0.0/16"
     noms_test_dr_vnet         = "10.111.0.0/16"
     noms_mgmt_dr_vnet         = "10.112.0.0/16"
+    corporate_staff_rostering_T3 = "10.0.0.0/8"
```

Comment:
This file is really for Azure specific IPs.  Our virtual networks etc.. .10.0.0.0/8 is just an RFC1918 IP, I don't think it belongs here

Reply:
Removed this from the file